components/speedreader/common/BUILD.gn

```diff
@@ -0,0 +1,14 @@
+# Copyright(c) 2021 The Brave Authors.All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0.If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import("//mojo/public/tools/bindings/mojom.gni")
+
+mojom("mojom") {
+  sources = [
+    "speedreader_result.mojom",
+    "speedreader_ui.mojom",
+  ]
+  deps = [ "//mojo/public/mojom/base" ]
```

Comment:
we use `public_deps`

> deps: DEPRECATED: This is synonymous with public_deps because all mojom dependencies must be public by design. Please use public_deps